examples/pruner/README.md

```diff
@@ -0,0 +1,29 @@
+# Pruning images using CronJob
+
+This example shows an image pruning happening in an automated fashion using the Kubernetes CronJobs that
+are available in OpenShift Origin from version 3.5.
+In this example, we will create a CronJob that will run image pruning every 15 minutes.
+
+## Requirements
+
+In order to execute the pruning commands successfully it is necessary to configure the
+authorization in a way that allows the `defaultq service account to perform the pruning
+against entire cluster (assuming you create the CronJob in the `default` project):
+
+1. `oc adm policy add-cluster-role-to-user sudoer admin-user --config=PATH/admin.kubeconfig`
+    This step is optional and it allows the "admin-user" to "sudo" into cluster
+    administrator role when needed.
```

Comment:
I'd drop this step and just require the steps to be run as cluster-admin. It unnecessarily complicates the steps.